addons/account/models/account_reconcile_model.py

```diff
@@ -106,6 +106,21 @@ class AccountReconcileModel(models.Model):
     second_analytic_account_id = fields.Many2one('account.analytic.account', string='Second Analytic Account', ondelete='set null')
     second_analytic_tag_ids = fields.Many2many('account.analytic.tag', string='Second Analytic Tags')
 
+    @api.multi
+    def action_reconcile_stat(self):
+        search_view_id = self.env.ref('account.view_account_move_line_filter').id
+        return {'name': 'Stat',
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'tree',
+            'res_model': 'account.move.line',
```

Comment:
open a tree view of account.move
(you need to add a function field for that (m2m))